sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaRulesDefinition.java

```diff
@@ -169,6 +170,17 @@ private void addMetadata(NewRule rule, @Nullable RuleMetadata metadata) {
       rule.setDebtRemediationFunction(metadata.remediation.remediationFunction(rule.debtRemediationFunctions()));
       rule.setGapDescription(metadata.remediation.linearDesc);
     }
+    if (supportsSecurityHotspots) {
+      addSecurityStandards(rule, metadata.securityStandards);
+    }
+  }
+
+  private static void addSecurityStandards(NewRule rule, SecurityStandards securityStandards) {
+    for (String s : securityStandards.OWASP) {
+      rule.addOwaspTop10(OwaspTop10.valueOf(s));
+    }
+    int[] cweNumbers = Arrays.stream(securityStandards.CWE).map(cwe -> cwe.substring(4)).mapToInt(Integer::valueOf).toArray();
```

Comment:
hm... do we want change API? to accept full names?

Reply:
this was discussed and decision was that they prefer to have int because it's simpler for them

Reply:
should rule api then produce numbers?

Reply:
done. rule-api was updated to use numbers